grails-app/views/order/_actions.gsp

```diff
@@ -94,7 +94,8 @@
 			</g:elseif>
 			<g:if test="${orderInstance?.orderType == PURCHASE_ORDER}">
 				<g:supports activityCode="${ActivityCode.PLACE_ORDER}">
-					<g:isSuperuser>
+					<g:isUserInRole roles="[RoleType.ROLE_SUPERUSER, RoleType.ROLE_ADMIN, RoleType.ROLE_ASSISTANT, RoleType.ROLE_MANAGER]">
+						<g:isSuperuser>
```

Comment:
1. Why are we using both isUserInRole and isSuperuser?
2. We only need to specify the lowest role ROLE_ASSISTANT.